Lesson2/src/Homework.java

```diff
@@ -0,0 +1,99 @@
+import java.util.Scanner;
+
+public class Homework {
+    public static void main(String[] args) {
```

Comment:
all solutions look good to me. Format the code, though. Press ctrl + alt + l to format code, please